src/py42/_compat.py

```diff
@@ -10,7 +10,7 @@
 is_py2 = _ver[0] == 2
 
 if is_py2:
-    from urllib import quote
+    from urllib import quote, urlencode
```

Comment:
This is not our formatters' preferred way to import multiple items from a module

Reply:
This file is ignored from formatting rules because it has a bunch of import that it will think are unused, but i can still change the way this import is done.